README.md

```diff
@@ -23,6 +23,7 @@ GitHub. Alternatively you can compile the plugin from source.
   ```sh
   $ mv vault-auth-plugin-example /etc/vault/plugins/vault-auth-plugin-example
   ```
+  Note that on MacOS, the etc directory is a symlink. Vault does not allow symlink directories for plugins.
```

Comment:
`/etc/` is also kind of a weird recommendation anyway. WDYT about just changing this command (and the SHA256 calculation below) to a user-defined variable? e.g. `$ mv vault-auth-plugin-example "$plugin_dir"`

Reply:
Although probably worth keeping this note in some form either way, as we can still mention `$plugin_dir` shouldn't contain any symlinks.